recipes/yaqd-core/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "yaqd-core" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "2d40037fb0506294cc82f879ffe0baeb7ed1c3c96572f6587e2e920ecd261fc3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - pytest-runner
```

Comment:
Usually you only need `python` and `pip` in the `host` section.

Reply:
It seems they do need ``pytest-runner`` in ``host`` :/
https://gitlab.com/yaq/yaqd-core-python/-/blob/master/setup.py#L24

Reply:
But you are completely right, usually just ``pip`` and ``python`` are enough